src/pages/studyView/StudyViewPage.tsx

```diff
@@ -83,16 +83,22 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
         this.store = new StudyViewPageStore();
 
         this.queryReaction = reaction(
-            () => props.routing.location.query,
-            query => {
+            () => [props.routing.location.query, props.routing.location.hash],
+            ([query,hash]) => {
 
                 if (!getBrowserWindow().globalStores.routing.location.pathname.includes("/study")) {
                     return;
                 }
 
                 this.store.updateCurrentTab(getStudyViewTabId(getBrowserWindow().globalStores.routing.location.pathname));
-                const newStudyViewFilter:StudyViewURLQuery = _.pick(props.routing.location.query, ['id', 'studyId', 'cancer_study_id', 'filters', 'filterAttributeId', 'filterValues']);
+                const newStudyViewFilter:StudyViewURLQuery = _.pick(query, ['id', 'studyId', 'cancer_study_id', 'filters', 'filterAttributeId', 'filterValues']);
 
+                if(hash) {
+                    const filters = hash.match(/filters=([^&]*)/);
```

Comment:
should we call that param filterJson to make ti super clear the format?

Reply:
updated